src/main/java/ch/securify/patterns/PatternDescription.java

```diff
@@ -25,20 +28,20 @@ public PatternDescription(String categoryId, Class patternId, String patternInfo
     enum Severity {
         Critical,
         High,
-        Medium,
-        Low
+        Medium
```

Comment:
@hiqua  Did you intentionally remove `Low` because it is currently not used?

Reply:
Yes.